### PR TITLE
Oss dev

### PR DESCRIPTION
**Pull request by cubap**
tada!

### CODE REVIEW

File: src/java/edu/slu/tpen/entity/image/Canvas.java [hunk 1]
```diff
@@ -187,6 +197,7 @@ public static JSONArray getLinesForProject(Integer projectID, String canvasID, I
         //annotationList.element("testing", "msid_creation");
         Transcription[] lines;
         lines = getProjectTranscriptions(projectID, folioNumber); //Can return an empty array now.
+        System.out.println(Arrays.toString(lines));
```
Comment (cubap) [inline src/java/edu/slu/tpen/entity/image/Canvas.java:200]: removing some of these system outs would be nicer...

**Review by thehabes — APPROVED** [review]
Did some of the comment cleanup, got rid of libs and files that should not be in the repo or tracked, updated gitignore

**Comment by thehabes** [comment]
Note this is deployed to https://dev.t-pen.org/TPEN/ for testing.  It is a full fledged production candidate.

**Comment by thehabes** [comment]
This has been deployed to production in connection with the DNS work with ITS @ SLU.  Functionality has been proven, and it has actually been running there for a while without reported issues.  

Ready for master.

**Pull request merged**